### DISCUSSION TITLE
Does oh-my-posh implement FTCS_COMMAND_START/EXECUTED/FINISHED shell integration sequences?

### DISCUSSION THREAD

**Discussion by rkeithhill** [JanDeDobbeleer/oh-my-posh#4966] [General]
I'm looking at this Window Terminal shell integration page: https://learn.microsoft.com/en-us/windows/terminal/tutorials/shell-integration

When I right-click in WT, I see the `Select output` menu item (cool!) but I don't see the `Select command` menu item (as shown in the last animated GIF): 
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/5177512/96e831b5-9eed-49c2-b131-4b303b91980b)

I don't see anything in the oh-my-posh prompt definition that uses the VT sequences:
```
`e]133;A or  `e]133;B or `e]133;C
```

**Comment by JanDeDobbeleer** [comment]
@rkeithhill did you enable shell integration in the configuration?

**Comment by rkeithhill** [comment]
No, thanks for the pointer.
